RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs

```diff
@@ -7,7 +7,14 @@
 namespace nl.SWEG.RPGWizardry.ResearchData
 {
     public class DataManager : MonoBehaviour
-    {
+    {
+        #region Inner Types
+        /// <summary>
+        /// Creating event array to check for spellunlocking
+        /// </summary>
+        public delegate void SpellUnlock();
```

Comment:
Single Responsibility Principle: It is not the DataManager's task to let other scripts know that a spell was unlocked. 
This functionality fits better in a script such as PlayerInventory.